src/trials.jl

```diff
@@ -344,6 +344,10 @@ function Base.show(io::IO, ::MIME"text/plain", t::Trial)
     print(io, "BenchmarkTools.Trial: ", length(t), " sample", if length(t) > 1 "s" else "" end,
           " with ", t.params.evals, " evaluation", if t.params.evals > 1 "s" else "" end ,".\n")
 
+    perm = sortperm(t.times)
+    times = t.times[perm]
+    gctimes = t.gctimes[perm]
```

Comment:
I'm not sure that we actually want `times` and `gctimes` to be paired like this. When we say "max gc time" I'd expect that to be the maximum GC time of the runs personally.

Reply:
So they should be sorted independently? Or gc times not sorted at all?

Reply:
See https://github.com/JuliaCI/BenchmarkTools.jl/pull/250/files#diff-4950e391a6cdecb0a6ab326a117041a5b97d914db33c6234d7f312a69790258cR360 they need to be sorted the same so that the calculation there makes sense.

Reply:
>  When we say "max gc time" I'd expect that to be the maximum GC time of the runs personally.

IIRC this has always been correlated e.g. we reported `gcratio(maximum(t))`.

Reply:
Hmm, I see.